### DISCUSSION TITLE
Git LFS disabled for account

### DISCUSSION THREAD

**Discussion by shrinidhin** [git-lfs/git-lfs#4521] [Q&A] [answered]
I've had trouble uploading my large files onto my git repository using Git LFS. There were a coupled of failed pushes to git lfs. Currently Git LFS is disabled for my account and it says that my storage has exceeded the limits. However, there are no files uploaded to my repository. Is there any way to release the storage?

**Comment by bk2204** [comment, marked as answer]
Hey,

This sounds like a problem with GitHub specifically, not the Git LFS client.  This repository is specifically for the client, which works with a variety of implementations, including GitHub.

The storage is for your account in total.  Because Git LFS uploads objects before the Git objects are pushed, it's possible that you do have some Git LFS objects uploaded even though your Git objects haven't been pushed.  If you have questions about what's been uploaded to GitHub or about your storage or bandwidth usage, I'd reach out to GitHub Support, who can help you with that.

**Reply by shrinidhin** [reply]
Ok.Thank you!
